apps/hash-api/src/graph/ontology/primitive/entity-type.ts

```diff
@@ -189,9 +247,10 @@ export const updateEntityType: ImpureGraphFunction<
     schema: ConstructEntityTypeParams;
 
     labelProperty?: BaseUrl;
+    instantiators: EntityTypeInstantiatorSubject[];
```

Comment:
Instantiators are tied to the versioned URL, not to the base URL, so it's required to specify it. On default arguments the reasoning from above applies as well.

Reply:
Why tied to the versioned URL, rather than the base URL? The base URL would seem more consistent with entities, where permissions are associated with the `baseId` (the `EntityId`) rather than an edition.

Once we introduce the ability to share editing permissions for a type with a specific user, we'll want this to apply to the type and future versions – otherwise every time a type is updated, we'll have to retrieve and apply all existing permissions on it to avoid changing them unintentionally.

Reply:
We currently use `VersionedUrl` everywhere for ontology type permission. I can change this to be the `BaseUrl` but that's a bigger change.

Reply:
Binding it to the `BaseUrl` implies that it's not possible to for example restrict older versions of a type to be instantiated.